src/main/kotlin/domain/racing/Cars.kt

```diff
@@ -0,0 +1,27 @@
+package domain.racing
+
+import util.racing.converter.PositionSymbolConverter
+import util.racing.generator.NumberGenerator
+import java.util.stream.IntStream
+import kotlin.streams.toList
+
+class Cars(private val cars: List<Car>) {
+
+    companion object {
+        fun of(carAmount: Int): Cars {
+            val cars = IntStream.range(0, carAmount)
+                .mapToObj { _ -> Car() }
```

Comment:
```suggestion
                .map { Car() }
```
map을 활용하셔도 좋을 것 같아요 🙂

Reply:
Int Steam을 변경하려고 하는것이기 때문에 map을 쓰면, 실제 Int로 변환해줘야 하는거 같아요.
그래서 MapToObj가 현재 상황에선 맞는거 같습니다.